Solutions/Marain.UserNotifications.Abstractions/Marain/Helpers/HashAlgorithmHelpers.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="HashAlgorithmHelpers.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Marain.Helpers
+{
+    using System.Security.Cryptography;
+    using System.Text;
+
+    /// <summary>
+    /// Hashing algorithms helper class.
+    /// </summary>
+    public static class HashAlgorithmHelpers
+    {
+        /// <summary>
+        /// Get hash.
+        /// </summary>
+        /// <param name="input">The input string which needs to be hashed.</param>
+        /// <returns>The byte array.</returns>
+        public static byte[] GetSHA256Hash(string input)
+        {
+            // Convert the input string to a byte array and compute the hash.
+            return SHA256.Create().ComputeHash(Encoding.UTF8.GetBytes(input));
```

Comment:
You previously had a `using` statement for the result of `SHA256.Create` but we've lost that in this refactoring.

Please can you add it back in to ensure that the `SHA256` instance it returns is properly disposed of.

Reply:
Hi Jon, Yes I guess so. I have added that back in and pushed the changes.